### ISSUE TITLE
How to display a Context Menu on a Mobile Device

### ISSUE THREAD

**Issue by swordream** [dotnet/maui#10730]
With the .NET MAUI in .NET 7 Release Candidate 1,We can now attach a context menu to any visual element using a MenuFlyout control. When the user right-clicks that view, the flyout will appear in that location when run on a desktop platform.
 
So,why not apply this feature to mobile device,for example,long tap the item to invoke the context menu, i think it's very useful。

**Comment by nebula2** [comment]
on mobile you typically use a swipeview

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/swipeview

**Comment by swordream** [comment]
> on mobile you typically use a swipeview
> 
> https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/swipeview

Yes, but sometimes the context menu is also very useful. WeChat uses a lot of context menus instead of the previous SwipeView。
